esp/src/eclwatch/WsDFUXref.js

```diff
@@ -0,0 +1,52 @@
+/*##############################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
```

Comment:
@fvillanustre Can you comment on what our policy on copyright messages in files needs to be?